yt_dlp/utils/_utils.py

```diff
@@ -5613,6 +5614,8 @@ def calculate_preference(self, format):
         if not format.get('tbr'):
             format['tbr'] = try_call(lambda: format['vbr'] + format['abr']) or None
 
+    def calculate_preference(self, format):
+        self._fill_sorting_fields(format)
```

Comment:
This is a bit awkward, but the function is public so we can keep compatibility this way